cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -24,29 +24,16 @@ using namespace IceInternal;
 string
 Slice::CsGenerator::getNamespacePrefix(const ContainedPtr& cont)
 {
-    //
```

Comment:
Simplified this loop.

`m` and `p` point to the same Slice element, the only difference is their typing.
But since all we do with `m` is call `getMetadataArgs` on it, we don't need this stricter typing.
(`getMetadataArgs` is defined on both `Contained` (`p`) and `Module` (`m`)).